src/FakeItEasy/Core/ArgumentConstraintTrap.cs

```diff
@@ -11,10 +11,12 @@ internal class ArgumentConstraintTrap
 
         public static void ReportTrappedConstraint(IArgumentConstraint constraint)
         {
-            if (trappedConstraints != null)
+            if (trappedConstraints == null)
             {
-                trappedConstraints.Add(constraint);
+                throw new InvalidOperationException("A.Ignored, A._ and A.That can only be used in the context of a call specification with A.CallTo()");
             }

```

Comment:
Super nitpick: I think we are in favour of the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma). I've been using it in comments and exception messages.